docs/pages/docs/configuration.mdx

```diff
@@ -0,0 +1,351 @@
+import Callout from 'components/Callout';
+import VersionTabs from 'components/VersionTabs';
+import {Tab} from 'nextra-theme-docs';
+
+# Global configuration
+
+Configuration properties that you commonly use across your Next.js app can be set globally.
+
+<Callout>
+  Note that if you're using [the Server Components
+  beta](http://localhost:3000/docs/getting-started/app-router-server-components),
+  you can set these properties in [the global request configuration
+  (`i18n.ts`)](http://localhost:3000/docs/getting-started/app-router-server-components#global-request-configuration)
+  to use them in Server Components.
+</Callout>
+
+## Time zone
+
+If possible, you should configure an explicit time zone, as this affects the rendering of dates and times. By default, the available time zone of the runtime will be used: In Node.js this is the time zone that is configured for the server and in the browser this is the local time zone of the user. As the time zone of the server and the one from the user can be different, this might be problematic when your app is both rendered on the server as well as on the client side.
+
+To ensure consistency, you can globally define a time zone:
```

Comment:
Suggestion: As a reader, I'm left wondering what to do in the case where my app spans multiple time zones. Perhaps consider adding a note hinting at a suggestion on how to handle such case.

Just an idea -- thanks for the lib!

Reply:
Thanks for chiming in!

Hmm, how do you handle this in your app?

Related to this, there's this note underneath:
> The time zone can either be static in your app, or alternatively read from the user profile if you store such a setting. The available time zone names can be looked up in [the tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).

Reply:
I've reworked the section a bit, maybe it's a bit clearer now: https://next-intl-docs-git-docs-app-router-first-next-intl.vercel.app/docs/configuration#time-zone

Let me know if this helps!